src/test/resources/pages/instructorFeedbackAddSuccess.html

```diff
@@ -5,6 +5,12 @@
     Edit Feedback Session
   </h1>
   <br>
+  <div data-session-creator="teammates.test1@gmail.tmt" id="session-creator-data">
+  </div>
+  <div data-students="{&quot;CFeedbackUiT.charlie.tmms@gmail.tmt&quot;:&quot;Team 2&quot;,&quot;CFeedbackUiT.danny.tmms@gmail.tmt&quot;:&quot;Team 2&quot;}" id="students-data">
+  </div>
+  <div data-instructors="[&quot;teammates.helper@gmail.tmt&quot;, &quot;teammates.test1@gmail.tmt&quot;, &quot;teammates.test2@gmail.tmt&quot;]" id="instructors-data">
```

Comment:
Hmm, since these are pure data anyway, perhaps we should just use the `<script>` and define javascript variables directly

Or maybe we can put these data attributes on the session/page related elements